orttraining/orttraining/python/ort_trainer.py

```diff
@@ -655,7 +601,7 @@ def __init__(self, model, loss_fn, model_desc, training_optimizer_name, map_opti
         # we use self.global_step_ to count optimizations being performed.
         # it is used to calculate learning rate if self.get_lr_this_step_ is provided.
         self.global_step_ = global_step
-        self.post_process_model_fn_ = postprocess_model
+        self.post_process_model_fn_ = _extra_postprocess
```

Comment:
I suggest renaming `post_process_model_fn_ ` for consistency with the public flag `_extra_postprocess`. Makes debugging easy